configure-slic/README.md

```diff
@@ -0,0 +1,70 @@
+# Configure slic
+
+The composite action prepares the environment for running the tests with [slic](https://github.com/stellarwp/slic).
+
+## Inputs
+
+| Name               | Required | Default | Description                                                                                                                                                                                                                                                                                   |
+|--------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `slic_ref`         | no       | 'main'  | Git reference to checkout slic repository. It might be needed for debug or testing purposes.                                                                                                                                                                                                  |
+| `php_version`      | no       | ''      | If it's not provided the [default slic PHP version](https://github.com/stellarwp/slic/blob/f89dbeaa7af9b795ede5d43be0fea3f8d929fd4a/.env.slic#L50) will be used. Only single dot notation is supported (e.g. 8.1, not 8.1.31).                                                                |
```

Comment:
Perhaps we should add a link to the [WordPress Supported Versions Chart](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/#supported-version-chart). For example, if you try PHP `8.3` with WordPress version `6.3`, it's not going to work.